### PR TITLE
Fix for JIRA issue APPM-1010

### PR DESCRIPTION
**Pull request by this**
Fixed by adding and updating necessary  .hbs and .js files.

### CODE REVIEW

File: features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/left-nav.js [hunk 1]
```diff
@@ -1,77 +1,95 @@
...
+	if (listPartial != 'view-mobileapp') {
+		return leftNavItems;
+	}
+
+	if (data.artifact == null) {
+		return leftNavItems;
+	}
+
+	var user = server.current(session);
+	var um = server.userManager(user.tenantId);
```
Comment (sajithab) [inline features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/left-nav.js:44]: Better to use meaningful variable name.

File: features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/view-mobileapp.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	return {
+		js: ['bootstrap-select.min.js'],
+		css: ['bootstrap-select.min.css']
+	};
+
+};
+
+var selectCategory = function (data) {
+	var selected,
+		arr = [],
```
Comment (sajithab) [inline features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/view-mobileapp.js:12]: arr? Meaningful name please.

File: features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/left-nav.js (hunk 1 above)
Comment (this) [inline features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/left-nav.js:44]: fixed in commit 11218c6c69c912675002976190e133aa70cf3914 of PR #186

File: features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/view-mobileapp.js (hunk 2 above)
Comment (this) [inline features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/helpers/view-mobileapp.js:12]: fixed in commit 70d3718c01995308c3e25734d68aafbbdca19d3b of PR #186

**Pull request merged**